AWSClientRuntime/Sources/Regions/DefaultRegionResolver.swift

```diff
@@ -11,7 +11,7 @@ public struct DefaultRegionResolver: RegionResolver {
     public let providers: [RegionProvider]
     let logger: SwiftLogger
 
-    public init(providers: [RegionProvider] = [EnvironmentRegionProvider(), ProfileRegionProvider(), IMDSRegionProvider()]) {
+    public init(providers: [RegionProvider] = [EnvironmentRegionProvider(), ProfileRegionProvider(), IMDSRegionProvider(), BundleRegionProvider()]) {
```

Comment:
can you actually move this one to be first in the list possibly? for iOS, all the other ones won't even work so I think it should be first. For linux it just will quickly pass over it.